client/web/web.go

```diff
@@ -380,6 +380,8 @@ func (s *Server) serveLoginAPI(w http.ResponseWriter, r *http.Request) {
 		s.serveTailscaleUp(w, r)
 	case r.URL.Path == "/api/device-details-click" && r.Method == httpm.POST:
 		s.serveDeviceDetailsClick(w, r)
+	case r.URL.Path == "/api/update/progress" && r.Method == httpm.GET:
+		s.serveGetUpdateProgress(w, r)
```

Comment:
Is this needed from the login client api? Thought updates really could only be triggered from the full management client